### ISSUE TITLE
Probleme d'affichage de langue pour LEXEAU

### ISSUE THREAD

**Issue by jonquet** [agroportal/project-management#847]
Cet affichage est incoherent: 

https://agroportal.lirmm.fr/ontologies/LEXEAU?p=classes&conceptid=https%3A%2F%2Fagroportal.lirmm.fr%2Fontologies%2FLEXEAU%23OWLClass_7396d21f_cd44_4be2_b9f4_11ea387ffa36&language=fr

<img width="709" alt="Capture d’écran 2024-06-06 à 15 22 44" src="https://github.com/agroportal/project-management/assets/7379183/421727d6-6400-4b9f-8dcc-ac747f8e88a3">

**Comment by Bilelkihal** [comment]
@jonquet 
Le problème c'est concernant le preferred name qui est en anglais?

**Comment by syphax-bouazzouni** [comment]
It is a backend issue,
![image](https://github.com/agroportal/project-management/assets/29259906/2c74c41d-5ed2-4435-bafa-9da76af7fe23)

The resource does not have prefLabels, it uses `rdfs:label`, so we generated one, using its` rdfs:labels` but we took only the English one.

**Comment by jonquet** [comment]
Indeed, this is a double bug: (i) we generate only one prefLabel for en and not 2 and (ii) we loose the language tag in doing so.

**Comment by jonquet** [comment]
Pour info, le problème concerne aussi les instances: 
https://agroportal.lirmm.fr/ontologies/LEXEAU?p=instances&instanceid=https%3A%2F%2Fagroportal.lirmm.fr%2Fontologies%2FLEXEAU%23OWLNamedIndividual_ef5183c1_af69_4c1d_b0c1_c1f0fb4596f2
